### DISCUSSION TITLE
Create programmatic flowchart

### DISCUSSION THREAD

**Discussion by aleresoftware** [clientIO/joint#1953] [Q&A]
Hi there,
I am wondering if I can use JointJs to make programmatic flowchart. The data will be in json format and based on the json data the flow will be determined. I tried searching on JointJs documentation, but I couldn't find. May be I missed it. Would you help to find the relevant documentation?

Thank you

**Comment by jamesgeorgewilliams** [comment]
JointJS provides the following two methods allowing you to convert the graph to a JSON object, and load a graph from a JSON object.
https://resources.jointjs.com/docs/jointjs/v3.6/joint.html#dia.Graph.prototype.toJSON
https://resources.jointjs.com/docs/jointjs/v3.6/joint.html#dia.Graph.prototype.fromJSON

A more detailed tutorial about these methods can be found here:
https://resources.jointjs.com/tutorial/serialization
